WebARonARKit/WebARonARKit/ViewController.m

```diff
@@ -217,16 +238,24 @@ - (void)refreshButtonClicked:(UIButton *)button {
 }
 
 - (void)setShowCameraFeed:(bool)show {
-    if (show) {
+    if (show && !USE_CAMERA_FRAME) {
         wkWebView.opaque = false;
         wkWebView.backgroundColor = [UIColor clearColor];
         wkWebView.scrollView.backgroundColor = [UIColor clearColor];
+        [wkWebView.scrollView
+            setContentInsetAdjustmentBehavior:
+            UIScrollViewContentInsetAdjustmentNever];
     } else {
         wkWebView.opaque = true;
         wkWebView.backgroundColor = wkWebViewOriginalBackgroundColor;
         wkWebView.scrollView.backgroundColor = wkWebViewOriginalBackgroundColor;
+        [wkWebView.scrollView
```

Comment:
same here - comment

Reply:
This was added to try to resolve a problem with the webview but I think it does not resolve it. I am waiting for a PR from @rezaali to land and I might remove this if it's not necessary. I will comment it at least.